components/faqs/ContributeFAQ.js

```diff
@@ -45,6 +45,21 @@ const ContributeFAQ = ({ collective, ...props }) => (
         </Content>
       </Entry>
     )}
+    {(collective.id !== collective.host.id || collective.id === collective.host.id) &&
```

Comment:
I added this condition `collective.id === collective.host.id` so the FAQ is displayed when contributing to the `fiscal host` directly too.